ep0099/doc.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that caan be found in the LICENSE file.
+
+// Package ep0099
```

Comment:
Package ep0099 controls a EP-0099 Raspberry Pi HAT with 4 relays via I²C.

or something like that.

Reply:
oops, yeah. my bad.